### ISSUE TITLE
react-northstar Toolbar menu item is missing aria-haspopup when it has a submenu

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluentui#17279]
### Environment Information

- **Package version(s)**: 0.52.1
- **Browser and OS versions**:  all

### Please provide a reproduction of the bug in a codepen:
https://fluentsite.z22.web.core.windows.net/0.52.1/components/toolbar/definition?showCode=false&showRtl=false&showTransparent=false&showVariables=false#content-menu-with-submenu

#### Actual behavior:

Item with submenu is missing `aria-haspopup={true}`

#### Expected behavior:

Item with submenu has `aria-haspopup={true}`

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)

Products/sites affected: (if applicable)
